.gitignore

```diff
@@ -170,8 +170,10 @@ go.work.sum
 main.jar
 
 # Secrets
-KEY
-gurobi.lic
+ampl_license_uuid*
+!ampl_license_uuid.template
+gurobi.lic*
+!gurobi.lic.template
```

Comment:
One could argue that these would be good candidates for a local .gitignore in each of the respective apps. This would avoid users accidentally adding their license to version control when copy&pasting these example apps.
However, just a thought and debatable design-wise. Not a blocking change for sure. :relaxed:

Reply:
No problem by me. I’ll add to local `.gitignore` files.

Reply:
Added the corresponding files to each project.